### PR TITLE
Featured projects

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
This is based on https://github.com/gtk-rs/gtk-rs.github.io/pull/229.

@sophie-h: I only kept the slick font (for the slider arrows).

### CODE REVIEW

File: _sass/_base.scss [hunk 1]
```diff
@@ -1,3 +1,12 @@
+@font-face {
+    font-family: 'slick';
+    font-weight: normal;
+    font-style: normal;
+
+    src: url('/font/slick.eot');
+    src: url('/font/slick.eot?#iefix') format('embedded-opentype'), url('/font/slick.woff') format('woff'), url('/font/slick.ttf') format('truetype'), url('/font/slick.svg#slick') format('svg');
```
Comment (sophie-h) [inline _sass/_base.scss:7]: I think the font itself is missing from the commit. And probably one format it enough? (I'm servering Cantarell only as woff.)

File: _sass/_layout.scss [hunk 2]
```diff
@@ -269,13 +291,78 @@ ul.contact {
     line-height: 1;
 }
 
+/**
+ * Carousel
```
Comment (sophie-h) [inline _sass/_layout.scss:295]: Is this still in use?

**Review by sophie-h — COMMENTED** [review]
There seems to be something of with the swipe. At the ends: To the left it jumps to the second. To the right it moves to empty page. Buttons work though.

**Comment by sophie-h** [comment]
This looks very nice!

File: _sass/_base.scss (hunk 1 above)
Comment (sophie-h) [reply _sass/_base.scss:7]: Yes, research says: just keep the woff.

File: _sass/_layout.scss (hunk 2 above)
Comment (GuillaumeGomez) [reply _sass/_layout.scss]: No indeed! Removed.

File: _sass/_base.scss (hunk 1 above)
Comment (GuillaumeGomez) [reply _sass/_base.scss]: Done. :)

**Comment by GuillaumeGomez** [comment]
So if we all agree with this, I'll squash commits so the extra files don't appear in the history.

**Comment by sophie-h** [comment]
We will have to check what to do with the screenshot size if we reduce the content width. But imo let's merge this first.

**Comment by GuillaumeGomez** [comment]
It's handled quite nicely. :)

**Comment by GuillaumeGomez** [comment]
Ok, time to test now if I didn't break everything. ;)

**Pull request merged**